app/helpers/accountants_helper.rb

```diff
@@ -0,0 +1,13 @@
+# Functions primarily related to accountant billing view
+module AccountantsHelper
+
+  def patientInitials(patient)
```

Comment:
Might behoove us to move this to the `Patient` object as an instance method, so we can do `@patient.initials`? thoughts?